### ISSUE TITLE
New release breaks Python 2.7 installations.

### ISSUE THREAD

**Issue by lukesneeringer** [google/protobuf#3042]
The release of protobuf today (I think within the last hour) seems to have broken my build on Python 2.7 only (works on 3.4 through 3.6).

https://travis-ci.org/googleapis/artman/jobs/228147078

I get tons of tracebacks that look like this:

```
Traceback:
test/test_goimport_task.py:18: in <module>
    from artman.tasks import protoc_tasks
artman/tasks/__init__.py:24: in <module>
    from artman.tasks import cleanup_tasks as cleanup
artman/tasks/cleanup_tasks.py:20: in <module>
    from artman.tasks import task_base
artman/tasks/task_base.py:23: in <module>
    from gcloud import logging as cloud_logging
.nox/unit_tests-python_version-2-7/lib/python2.7/site-packages/gcloud/logging/__init__.py:17: in <module>
    from gcloud.logging.client import Client
.nox/unit_tests-python_version-2-7/lib/python2.7/site-packages/gcloud/logging/client.py:37: in <module>
    from gcloud.client import JSONClient
.nox/unit_tests-python_version-2-7/lib/python2.7/site-packages/gcloud/client.py:20: in <module>
    from gcloud._helpers import _determine_default_project
.nox/unit_tests-python_version-2-7/lib/python2.7/site-packages/gcloud/_helpers.py:28: in <module>
    from google.protobuf import timestamp_pb2
.nox/unit_tests-python_version-2-7/lib/python2.7/site-packages/google/protobuf/timestamp_pb2.py:6: in <module>
    from google.protobuf import descriptor as _descriptor
.nox/unit_tests-python_version-2-7/lib/python2.7/site-packages/google/protobuf/descriptor.py:46: in <module>
    from google.protobuf.pyext import _message
E   ImportError: libprotobuf.so.13: cannot open shared object file: No such file or directory
```

**Comment by xfxyjwf** [comment]
How is protobuf installed in your test? Does your test just grab the latest version of protobuf?

**Comment by xfxyjwf** [comment]
right now the protobuf python package I pushed to PyPI is the pure python version, so I'm not sure why it tries to reference libprotobuf.so.13 in your test.

**Comment by lukesneeringer** [comment]
It grabs the latest version that is `>=3.2.0, < 4.0.0dev`. So, it opts in to semver minor or patch bumps, but not major ones.

**Comment by xfxyjwf** [comment]
I removed protobuf-3.3.0-py2.py3-none-any.whl from PyPI. Can you rerun the test and see if it makes a difference?

**Comment by lukesneeringer** [comment]
At the moment, I am getting the same error. It still works in Python 3.

**Comment by xfxyjwf** [comment]
Can you try again? I removed another protobuf-3.3.0-py2.7.egg file.

**Comment by xfxyjwf** [comment]
I intended to deploy a pure python version but it seems "python setup.py bdsit_egg bdist_wheel" will actually include lots of temporary files in my workspace and that happens to add "_message.so" to the package. That's the only reason I can think of. Not sure why it works for python 3.x but not python 2.7 though.

**Comment by tedivm** [comment]
This broke for us too. We downgraded back to 3.2.0 and it fixed itself. It appears that rather than use the pure python version it's looking for the shared library.

**Comment by lukesneeringer** [comment]
Fixed: https://travis-ci.org/googleapis/artman/jobs/228147078

**Comment by xfxyjwf** [comment]
@tedivm Can you try again with 3.3.0?

**Comment by xfxyjwf** [comment]
Closing as the cloud logging issue is fixed.  @tedivm Please open a new issue if you still observe problems with the new release.

**Comment by dhermes** [comment]
@xfxyjwf More confirmation that you have indeed fixed the `3.3.0` release:

- [First broken build][2]
- [Second broken build][2]
- [Newly good build][1]

[1]: https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/1729
[2]: https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/1726
[3]: https://circleci.com/gh/GoogleCloudPlatform/google-cloud-python/1727

**Comment by tedivm** [comment]
I can also confirm that the new package works.
